### DISCUSSION TITLE
tetrahedron labilization

### DISCUSSION THREAD

**Discussion by J-AKAZ** [neperfepx/neper#133] [Q&A] [answered]
Hello dears,
I would like to know if there is a way on Neper to provide an ouput file containing the number of the tetrahedron and the number of the grain to which it belongs?
Thank you :)

**Comment by rquey** [comment, marked as answer]
`-statelt elset3d`

**Reply by J-AKAZ** [reply]
Thank you for your answer. But this command just gives the grain numbers and not the element numbers (tetrahedron) could you help me more please?

**Reply by rquey** [reply]
Okay, in that case: `-statelt id,elset3d`.
